proto/replicate_mode.proto

```diff
@@ -0,0 +1,35 @@
+syntax = "proto3";
+package replicate_mode;
+
+// The replicate status sync from PD to TiKV.
+message ReplicateStatus {
```

Comment:
Maybe we need to name it as `ReplicationStatus`?

Reply:
I think it ok. You can send a PR.